themes/classic/_dev/css/components/imageslider.scss

```
@@ -22,6 +22,28 @@
       max-width: 340px;
       .caption-description p {
         color: white;
+        @media (max-width: 767px) {
```

You should use bootstrap's includes:
```suggestion
        @include media-breakpoint-down(md) {
```

But watch out, `md` is 768px

---

Further is this document we can read:

```
@include media-breakpoint-down(md) {
  .carousel {
    .carousel-item {

[...]

      .caption-description {
        display: none;
      }
    }
  }
}
```

Modifying its size is a non sense as it is anyway hidden.